cli/commands/team23.go

```diff
@@ -0,0 +1,51 @@
+/*
+Root command is created as team23. Holds no functionality but all other
```

Comment:
Might want to update this

Reply:
Would you choose a different name? This root command still has to exist and when it gets executed, the checking of file path is done or rest of commands.

Reply:
Should have clarified, just mean the part "Holds no functionality.." since we now use it for the `./run URL_FILE` command